service/history/handler.go

```diff
@@ -1575,6 +1572,25 @@ func (h *Handler) ReapplyEvents(
 	return nil
 }
 
+// GetReplicationTasksFromDLQ is only used for integration test verification purpose
+func (h *Handler) GetReplicationTasksFromDLQ(sourceClusterName string) ([]*persistence.ReplicationTaskInfo, error) {
+	var tasksFromDLQ []*persistence.ReplicationTaskInfo
+	for i := 0; i < h.config.NumberOfShards; i++ {
+		shardItem := h.controller.getHistoryShardItem(i)
+		if shardItem != nil {
```

Comment:
if shard item is not present in this host, then just giveup?
i know this API is for integration test, but why creating an API just for test?

Reply:
Removed this method.